package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "ucw-app",
   "version": "0.1.0",
-  "private": true,
   "scripts": {
-    "server": "npx ts-node server/server.js",
+    "server": "ts-node server/server.js",
+    "ui": "cd ./ui && npm run dev",
```

Comment:
This feels brittle. Wouldn't this fail if you ran it twice, because you would have already changed folders?

Reply:
Yeah, good point. I added this for use in the Dockerfile, but I'll figure out a different way to do this.

Reply:
I ended up splitting the UI app into its own docker container, and was able to get rid of this.